src/tasks/batch_integration/metrics/hvg_overlap/script.py

```diff
@@ -13,27 +13,24 @@
 ## VIASH END
 
 print('Read input', flush=True)
-adata = ad.read_h5ad(par['input_integrated'])
-
-print('prepare data')
-adata_unint = adata.copy()
-adata_unint.X = adata_unint.layers["normalized"]
-adata.X = adata.layers["corrected_counts"]
-
-print('compute score')
+input_solution = ad.read_h5ad(par['input_solution'])
+input_integrated = ad.read_h5ad(par['input_integrated'])
+input_solution.X = input_solution.layers["normalized"]
+input_integrated.X = input_integrated.layers["corrected_counts"]
 
+print('compute score', flush=True)
 score = hvg_overlap(
```

Comment:
Can a 'layer' argument be added to the `hvg_overlap` function? This would save us the trouble of having to copy the normalized data to `.X`.

Reply:
Alright, thanks. When the time comes, let's do this in a separate PR.